bitcoin/src/blockdata/transaction.rs

```diff
@@ -117,7 +116,7 @@ impl OutPoint {
     /// The `OutPoint` used in a coinbase prevout.
     ///
     /// This is used as the dummy input for coinbase transactions because they don't have any
-    /// previous outputs. This is not a real outpoint and should not be used in any other contexts.
+    /// previous outputs. In other words, does not point to a real transaction.
```

Comment:
Also "should not be used in other contexts" should stay.

Reply:
I'm confused, this change from ea2efc155e1b62140298382a874ad4355a5b3d97 which merged already in #3308, I've no idea how its showing up here.

Reply:
Interesting, I'm confused how I didn't notice it. Maybe it merged before I understood the issues around these things (esp. wTXID).